pre_commit/main.py

```diff
@@ -67,6 +85,14 @@ def main(argv=None):
     run_parser.add_argument(
         '--verbose', '-v', action='store_true', default=False,
     )
+    run_parser.add_argument(
+        '--origin', '-o', default='',
+        help='The origin branch/commit_id when use `git push`',
+    )
+    run_parser.add_argument(
+        '--source', '-s', default='',
```

Comment:
I think this is necessary, because there are other options. they are not required